tests/e2e/image-spec.yaml

```diff
@@ -127,6 +127,8 @@ build: |
       && make -j $(nproc) \
       && make install
 
+  FROM quay.io/curl/curl-base:8.12.1 AS curl
```

Comment:
why is that? I see that curl is installed with apt on line 150 in the same file

Reply:
Removed. It was a leftover from my earlier attempt to get 'curl' from the docker image.